src/help/components/entry/EntryPreview.tsx

```diff
@@ -0,0 +1,183 @@
+        content,
+        lastModified: new Date().toISOString(),
+      },
+      otherAttributes,
+    ];
+  }, [data]);
+
+  const handleChange = useCallback((event: ChangeEvent<HTMLInputElement>) => {
+    const url = event.target.value.trim();
+    if (!url || !url.startsWith('https://')) {
+      return;
+    }
+    if (url.startsWith('https://github.com/')) {
+      // we don't have the raw file, try to find it
+      const ghWikiMatch = url.match(ghWikiRE);
+      if (ghWikiMatch?.groups) {
+        setUrl(
+          `https://raw.githubusercontent.com/wiki/${ghWikiMatch.groups.user}/${ghWikiMatch.groups.project}/${ghWikiMatch.groups.file}.md`
+        );
+        return;
+      }
+
+      const ghNormalMatch = url.match(ghNormalRE);
+      if (ghNormalMatch?.groups) {
+        setUrl(
+          `https://raw.githubusercontent.com/${ghNormalMatch.groups.user}/${ghNormalMatch.groups.project}/${ghNormalMatch.groups.filepath}`
+        );
+        return;
+      }
+    }
+    setUrl(url);
+  }, []);
+
+  // Note: copy structure of "normal" help entry page in order to preview it
+  // Maybe we should just use the actual help entry page?
+  return (
+    <>
+      <HTMLHead>
+        {/* Don't index 4xx pages */}
+        <meta name="robots" content="noindex" />
+      </HTMLHead>
+      <section className={styles['private-area']}>
+        <div>The content outside of this area is a preview</div>
+        <label>
+          Load article content from a URL:{' '}
+          <input type="url" onChange={handleChange} />
+        </label>
+        {data && (
+          <output>
+            <details>
+              <summary>Raw loaded content</summary>
+              <CodeBlock>{data}</CodeBlock>
+            </details>
+          </output>
+        )}
+        {data && (
+          <output>
+            <details>
+              <summary>Parsed metadata</summary>
+              <dl>
+                <dt>
+                  title{' '}
+                  {parsedData.title ? '✅' : '❌ a title needs to be provided'}
+                </dt>
+                <dd>{parsedData.title}</dd>
+              </dl>
+              <dl>
+                <dt>
+                  categories{' '}
+                  {parsedData.categories.length
+                    ? '✅'
+                    : '❌ one or more categories need to be provided'}
+                </dt>
+                <dd>
+                  {parsedData.categories.map((category) => (
+                    <Category category={category} key={category} />
+                  ))}
+                </dd>
+              </dl>
+              {Object.entries(otherAttributes || {}).map(([key, value]) => (
+                <dl>
+                  <dt>{key} ❌ ignored/invalid field</dt>
+                  <dd>{value}</dd>
+                </dl>
+              ))}
+            </details>
+          </output>
+        )}
+      </section>
+      <HelpEntry {...props} overrideContent={parsedData} />
```

Comment:
```suggestion
      {parsedData !== defaultData && (
        <HelpEntry {...props} overrideContent={parsedData} />
      )}
```
Just draws the eyes more to the url on initial load